.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: Rails Template CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # * At 05:00 UTC every Monday, run the latest commit on the default or base branch
+    - cron:  '0 5 * * MON'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      # don't stop all variants if one of them fails (we usually want to know
+      # how many fail)
+      fail-fast: false
+      matrix:
```

Comment:
Would be good to also test across the three OSs

Reply:
(for Windows, I'd be happy to test against WSL rather than trying to get it working in native)

Reply:
Hmmm that's a good idea but I'd prefer not to block this PR on adding that (it's not something we currently do on TravisCI). I'll make an issue to track it.

Reply:
Yeah, happy with tracking this via #122 to avoid blocking :)